packages/alert-message-react/src/AlertMessage.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import classNames from "classnames";
+import { MessageIcon } from "./common/MessageIcon";
+
+type messageTypes = "info" | "error" | "success" | "warning";
+
+interface Props {
+    children: React.ReactNode;
+    className?: string;
+    inverted?: boolean;
+    maxWidth?: string;
```

Comment:
Lurer på om jeg ville kalt denne `maxContentWidth` eller noe i stedet? Bare for å understreke at selve meldingsboksen uansett er fullbredde.

Reply:
Ja, det er lurt.

Reply:
Æsj, nå husker jeg hvorfor den het maxWidth. Den mappet direkte mot CSS-navnet 🤦 Borky bork